gsa/src/web/pages/nvts/listpage.js

```diff
@@ -50,50 +72,195 @@ export const ToolBarIcons = () => (
   />
 );
 
-const Page = ({filter, onFilterChanged, onInteraction, ...props}) => (
-  <React.Fragment>
-    <PageTitle title={_('NVTs')} />
-    <EntitiesPage
-      {...props}
-      createFilterType="info"
-      dashboard={() => (
-        <NvtsDashboard
+const fallbackFilter = Filter.fromString('sort-reverse=created');
+
+const NvtsPage = () => {
+  // Page methods and hooks
+  const [downloadRef, handleDownload] = useDownload();
+  const [, renewSessionTimeout] = useUserSessionTimeout();
+
+  // Powerfilter
+  const [filter, isLoadingFilter] = usePageFilter('nvt', {fallbackFilter});
+  const prevFilter = usePrevious(filter);
+  const simpleFilter = filter.withoutView();
+  const {
+    change: changeFilter,
+    remove: removeFilter,
+    reset: resetFilter,
+  } = useChangeFilter('nvt');
+  const {
+    dialogState: notificationDialogState,
+    closeDialog: closeNotificationDialog,
+    showError,
+  } = useDialogNotification();
+  const {
+    selectionType,
+    selected = [],
+    changeSelectionType,
+    select,
+    deselect,
+  } = useSelection();
+  const [sortBy, sortDir, handleSortChange] = useFilterSortBy(
+    filter,
+    changeFilter,
+  );
+  const [tagsDialogVisible, setTagsDialogVisible] = useState(false);
+
+  // Nvt list state variables and methods
+  const [
+    getNvts,
+    {counts, nvts, error, loading: isLoading, refetch, called, pageInfo},
+  ] = useLazyGetNvts();
+
+  const exportNvtsByFilter = useExportNvtsByFilter();
+  const exportNvtsByIds = useExportNvtsByIds();
+  const bulkExportNvts = useBulkExportEntities();
+
+  const timeoutFunc = useEntitiesReloadInterval(nvts);
+
+  const [startReload, stopReload, hasRunningTimer] = useReload(
+    refetch,
+    timeoutFunc,
+  );
+
+  // Pagination methods
+  const [getFirst, getLast, getNext, getPrevious] = usePagination({
+    simpleFilter,
+    filter,
+    pageInfo,
+    refetch,
+  });
+
+  // No Nvt methods ?
```

Comment:
The methods are for the actions that are available on the page. Since there are no actions on the nvt listpage (except for bulk actions) we don't need any methods here.

Reply:
```suggestion
```